api/cloudcontroller/ccv2/constant/event.go

```diff
@@ -0,0 +1,27 @@
+package constant
+
+// EventType is the type of event that occurred in cloud foundry.
+type EventType string
+
+const (
```

Comment:
Do we have the necessary info for adding documentation comments for these new constants?

Reply:
I couldn't find any official documentation from CAPI about these event types, but I can add some Godocs based on my experiments with the API.

Reply:
Given that there's no official docs from CAPI and the fact that these type names are pretty self-evident, I think that it's fine to forego adding the godoc lines here. 

@XenoPhex wdyt?

Reply:
@tjvman @XenoPhex 
Should I write out all the events? Personally, I forsee use cases for about half a dozen more s it might make sense to list out all the valid event types right now while I'm at it (https://apidocs.cloudfoundry.org/280/events/list_all_events.html). 

Currently I'm using one of the experimental event types. I can add or leave out all the experimental types, whichever you prefer?

Reply:
If you want me to add Godoc I came up with the following descriptions:
`// EventTypeAuditApplicationCreate denotes an event where the CC creates an applicaion.`
`// EventTypeAuditApplicationDeleteRequest denotes an event where the CC receives a request to delete an application.`

Reply:
Ideally you'd add all of them with [some kind of] documentation, but for this specific case, that seems excessive.

I'd say at minimum, make a constant for each but only add GoDoc to the constants you know about. We'll assume anything you leave blank will require us to figure it out. If that ends up being most of them, that's a'ok.